consensus/core/src/metrics.rs

```diff
@@ -692,6 +693,11 @@ impl NodeMetrics {
                 &["authority", "error"],
                 registry
             ).unwrap(),
+            fetch_commits_skip_uncertified: register_int_counter_with_registry!(
+                "fetch_commits_skip_uncertified",
+                "Number of uncertified commits that got skipped to return when fetching commits",
```

Comment:
nit: "Number of uncertified commits that got skipped when fetching commits due to lack of votes"